### PR TITLE
fix: count in statistics overflow

### LINKED ISSUE #3001: The result of monitor is negative
WX20230130-193022@2x
WX20230130-193454@2x

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by aceforeverd** [comment]
1. `statistics::TimeCollector::count_`: uint32 -> uint64
2. at least a warning message should produce on nameservr init, inside `CreateSystemTableOrExit(kDeployResponseTime)`, where if old system table exists

**Comment by aceforeverd** [comment]
data type in exporter might change too, e.g https://github.com/4paradigm/OpenMLDB/blob/48f239a84c587a8586c7132ecff32fdf7c5a39c0/monitoring/openmldb_exporter/collector/collectors.py#L100

File: src/nameserver/name_server_impl.cc [hunk 1]
```diff
@@ -10789,7 +10789,12 @@ void NameServerImpl::SyncDeployStats() {
     while (rs->Next()) {
         auto name = rs->GetAsStringUnsafe(0);
         auto time = rs->GetAsStringUnsafe(1);
-        int32_t cnt = rs->GetInt32Unsafe(2);
+        int64_t cnt = 0;
+        if (rs->GetSchema()->GetColumnType(2) == hybridse::sdk::DataType::kTypeInt64) {
+            cnt = rs->GetInt64Unsafe(2);
```
Comment (aceforeverd) [inline src/nameserver/name_server_impl.cc]: better cast to uint64 explictly

File: src/nameserver/name_server_impl.cc [hunk 2]
```diff
@@ -5572,8 +5572,12 @@ void NameServerImpl::OnLocked() {
         CreateSystemTableOrExit(SystemTableType::kGlobalVariable);
         InitGlobalVarTable();
     }
-    if (FLAGS_system_table_replica_num > 0 && db_table_info_[INFORMATION_SCHEMA_DB].count(DEPLOY_RESPONSE_TIME) == 0) {
-        CreateSystemTableOrExit(SystemTableType::kDeployResponseTime);
+    if (FLAGS_system_table_replica_num > 0) {
+        if (db_table_info_[INFORMATION_SCHEMA_DB].count(DEPLOY_RESPONSE_TIME) == 0) {
+            CreateSystemTableOrExit(SystemTableType::kDeployResponseTime);
```
Comment (aceforeverd) [inline src/nameserver/name_server_impl.cc:5578]: `..OrExist` already handles if table exist. My concern is the user upgrade from old version and `deploy_response_time` already exists with the `(string, string,  int, string)` schema, we won't correct the schema. So at least, we should detect the old table, and print a warning log about the potential count overflow

**Review by aceforeverd — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

File: monitoring/openmldb_exporter/collector/collectors.py [hunk 3]
```diff
@@ -64,11 +64,11 @@ def collect(self):
             tid, tb_name, db_name, storage_type, rows, mem, disk, partition, partition_unalive, replica, *_ = row
             tb_path = f"{db_name}_{tb_name}"
             tid = int(tid)
-            table_rows.labels(tb_path, tid, storage_type).set(int(rows))
+            table_rows.labels(tb_path, tid, storage_type).set(long(rows))
```
Comment (aceforeverd) [inline monitoring/openmldb_exporter/collector/collectors.py:67]: There is no `long` for Python

**Pull request merged**